components/slider/index.css

```
@@ -133,49 +156,48 @@ governing permissions and limitations under the License.
 .spectrum-Slider-track {
   &:first-of-type {
     &:before {
-      border-start-start-radius: var(--spectrum-slider-track-border-radius);
-      border-end-start-radius: var(--spectrum-slider-track-border-radius);
-      border-start-end-radius: 0;
-      border-end-end-radius: 0;
+      border-start-start-radius: var(--mod-slider-track-corner-radius, var(--spectrum-slider-track-corner-radius));
+      border-end-start-radius: var(--mod-slider-track-corner-radius, var(--spectrum-slider-track-corner-radius));
     }
   }
 
   &:last-of-type {
     &:before {
-      border-start-start-radius: 0;
-      border-end-start-radius: 0;
-      border-start-end-radius: var(--spectrum-slider-track-border-radius);
-      border-end-end-radius: var(--spectrum-slider-track-border-radius);
+      border-start-end-radius: var(--mod-slider-track-corner-radius, var(--spectrum-slider-track-corner-radius));
+      border-end-end-radius: var(--mod-slider-track-corner-radius, var(--spectrum-slider-track-corner-radius));
     }
   }
 }
 
-.spectrum-Slider-track ~ .spectrum-Slider-track {
+.spectrum-Slider-track~.spectrum-Slider-track {
   inset-inline-start: auto;
-  inset-inline-end: var(--spectrum-slider-range-track-reset);
+  inset-inline-end: var(--mod-slider-range-track-reset, var(--spectrum-slider-range-track-reset));
   padding-block: 0;
-  padding-inline: var(--spectrum-slider-track-handleoffset) 0;
-  margin-inline-start: var(--spectrum-slider-range-track-reset);
-  margin-inline-end: var(--spectrum-slider-track-margin-offset);
+  padding-inline: var(--mod-slider-track-handleoffset, var(--spectrum-slider-track-handleoffset)) 0;
```

```suggestion
  padding-inline-start: var(--mod-slider-track-handleoffset, var(--spectrum-slider-track-handleoffset));
  padding-inline-end: 0;
```

That little zero at the end is sneaky - should we pull it out into distinctly named properties so we don't miss it when debugging?